Corona-Warn-App/src/main/res/values-de/legal_strings.xml

```diff
@@ -34,4 +34,6 @@
     <string name="submission_your_consent_agreement_share_test_results">"<b>Wenn Sie positiv auf Corona getestet wurden, teilt die App Ihr Testergebnis, um Nutzer zu warnen, denen Sie begegnet sind.</b>\n\n<b>Die Warnung erreicht Nutzer von offiziellen Corona-Apps der folgenden teilnehmenden Länder:</b>"</string>
     <!-- YTXT: Body for your consent screen agreement share symptoms -->
     <string name="submission_your_consent_agreement_share_symptoms">"<b>Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b>\n\nSie können Ihr Einverständnis zurücknehmen, indem Sie oben „Andere warnen“ deaktivieren."</string>
+    <!-- YTXT: Body for keys submission no consent text -->
+    <string name="submission_no_consent_your_consent_subsection_body">"Durch Antippen von „Einverstanden“ willigen Sie wie folgt ein: \n\n<b>Die App teilt Ihr Testergebnis, um Nutzer von offiziellen Corona-Apps, denen Sie begegnet sind, zu warnen. Die Warnung funktioniert in den oben genannten Ländern. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b> Bitte geben Sie Ihr Testergebnis (genauer gesagt: Ihrer Zufalls-IDs der letzten 14 Tage) im nächsten Schritt frei, indem Sie „Teilen“ antippen.\n\nSie können Ihr Einverständnis nach bereits erfolgter Warnung zurücknehmen, indem Sie die App löschen."</string>
```

Comment:
There are too many line breaks, the space between paragraphs is too big.

Reply:
what do you recon here then? seperate texts for each line?

Reply:
I think line breaks are okay. We are using them in many screens for now.